--- install.sh	2015-05-18 14:28:36.731513736 +0000
+++ install-mod.sh	2015-05-18 14:31:26.227261377 +0000
@@ -153,26 +153,6 @@
       $SYSTEM_LOCAL_SHARE/man/man1/renoise.1.gz || bail_out
     install -D -m644 $SCRIPT_PATH/Installer/renoise-pattern-effects.5.gz \
       $SYSTEM_LOCAL_SHARE/man/man5/renoise-pattern-effects.5.gz || bail_out
-    
-    echo "Registering MIME types..."
-    xdg-mime install --novendor $RESOURCES_PATH/Installer/renoise.xml || \
-      (echo "  Warning: failed to register the Renoise MIME types (not critical)."; \
-       echo "  Your desktop environment might not support the installation of MIME types...\n")
-  
-    echo "Installing icons..."
-    (xdg-icon-resource install --novendor --size 48 --context apps \
-       $RESOURCES_PATH/Installer/renoise.png && \
-     xdg-icon-resource install --novendor --size 48 --context mimetypes \
-       $RESOURCES_PATH/Installer/renoise.png application-x-renoise-module && \
-     xdg-icon-resource install --novendor --size 48 --context mimetypes \
-       $RESOURCES_PATH/Installer/renoise.png application-x-renoise-rns-module) || \
-      (echo "  Warning: failed to install the Renoise icon files (not critical)."; \
-       echo "  Your desktop environment might not support the installation of app icon files...\n")
-
-    echo "Installing desktop-menu shortcuts..."
-    xdg-desktop-menu install --novendor $RESOURCES_PATH/Installer/renoise.desktop || \
-      (echo "  Warning: failed to install desktop-menu shortcuts for Renoise (not critical)."; \
-       echo "  Your desktop environment might not support the installation of app shortcuts...\n")
   fi  
   
   if [ $INSTALLATION_MODE -eq $INSTALL ]; then
@@ -200,5 +180,4 @@
   fi  
 fi
 
-exit 1
-
+exit 0
